docs/how-we-work/tools/text-editors.md

```diff
@@ -1,17 +0,0 @@
-# Text editors
```

Comment:
I am not sure why this was moved - text editors seem more like tools than practices to me